.github/workflows/greetings.yaml

```
@@ -0,0 +1,19 @@
+name: First Time Contributor Greetings
+
+on:
+  pull_request:
+    types: [opened]
+    branches:
+      - develop
+
+jobs:
+  greeting:
+    if: "github.repository == 'MDAnalysis/mdanalysis'"
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+    steps:
+    - uses: actions/first-interaction@v1
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        pr-message: 'Hello there first time contributor! Welcome to the MDAnalysis community! We ask that all contributors abide by our [Code of Conduct](https://www.mdanalysis.org/pages/conduct/) and that first time contributors introduce themselves on the [developer mailing list](https://groups.google.com/g/mdnalysis-devel) so we can get to know you. You can learn more about participating [here](https://www.mdanalysis.org/#participating)'
```

The following was in one of my previous comments:
```suggestion
        pr-message: 'Hello there first time contributor! Welcome to the MDAnalysis community! We ask that all contributors abide by our [Code of Conduct](https://www.mdanalysis.org/pages/conduct/) and that first time contributors introduce themselves on the [developer mailing list](https://groups.google.com/g/mdnalysis-devel) so we can get to know you. You can learn more about participating [here](https://www.mdanalysis.org/#participating). Please also add yourself to `package/AUTHORS` as part of this PR.'
```

Feel free to ignore if it makes the text too verbose or you do not deem it usefil.

---

I like the additional sentence!

---

I am not too fond of "here" links but I can't think of anything substantially better. Maybe put "participation" inside the link anchor, too.